test/Chirp.Infrastructure.Tests/RestrictedCheepsUnitTests.cs

```diff
@@ -126,6 +126,6 @@ public void TestRestrictedCheepIsAddedToMemoryDatabase()
         cheeps.Should().NotBeNull(); //Makes sure the page is not empty
 
         //Assert
-        cheeps.Should().Contain(c => c.AuthorId == 1 && c.Message == "This is a cheep for testing");
+        cheeps.Should().Contain(c => c.CheepId == new Guid(1, 0, 0, new byte[]{0,0,0,0,0,0,0,0}) && c.Message == "This is a cheep for testing");
```

Comment:
I think that this is not the way you use Guid. You should use Guid.NewGuid() instead of new Guid().